src/routes/UnauthenticatedRoute/UnauthenticatedRoute.tsx

```diff
@@ -15,16 +17,18 @@ export const UnauthenticatedRoute = () => {
 
   return (
     <Suspense fallback={<Loading />}>
-      <Routes>
-        <Route index element={<Navigate to="/logout/user" replace />} />
-        <Route path="/login" element={<Login />} />
-        <Route path="/registration" element={<Registration />} />
-        <Route path="/confirmotp" element={<ConfirmOTP />} />
-        <Route path="/resetpassword-phone" element={<ResetPasswordPhone />} />
-        <Route path="/resetpassword-confirmotp" element={<ResetPasswordConfirmOTP />} />
-        <Route path="/organization-registration" element={<OrganizationRegistration />} />
-        <Route path="/*" element={<Navigate to="/logout/user" replace />} />
-      </Routes>
+      <GoogleReCaptchaProvider reCaptchaKey={RECAPTCHA_CLIENT_KEY}>
```

Comment:
@mdshamoon 

We have added Recaptcha for all non-authenticated routes but not validating the same.

For eg: Recaptcha is shown on login and forgot password but there is no validation for the same.

Secondly, I think we need to handle the scenario when Recaptcha expires on the page. I think backed should provide appropriate error message and frontend can display the same (like refresh page)

Reply:
Based on our discussion I have updated the PR with the error message from the backend and keeping the catcha script for all non authenticated routes.